.circleci/config.yml

```diff
@@ -118,15 +118,17 @@ jobs:
 
   test:
     <<: *container_config
+    parallelism: 4
     steps:
       - *restore_repo
       - *sum_lockfiles
       - *restore_npm_cache
       - *npm_install
       - run:
           name: "Run tests with jest"
-          command:
-            npm test -- --runInBand --no-cache --coverage --reporters=jest-junit
+          command: |
+            TEST=$(circleci tests glob **/packages/*/src/**/*.test.tsx | circleci tests split --split-by=timings)
```

Comment:
the `glob **/packages/*/src/**/*.test.tsx` is to avoid circle from including `node_modules`. I tried the `!(node_modules)` but I can't seem to get it working in circle.